include/xtensor/xstrided_view.hpp

```diff
@@ -400,10 +403,26 @@ namespace xt
         return *this;
     }
 
+    namespace xstrided_view_detail
+    {
+        template <class V, class T>
+        inline void run_assign_temporary_impl(V& v, const T& t, std::true_type)
```

Comment:
Could you add a commented name for the last parameter? `std::true_type` is not really meaningful